### DISCUSSION TITLE
Please improved RGB control

### DISCUSSION THREAD

**Discussion by xavier-phillips** [4JX/L5P-Keyboard-RGB#125] [Ideas]
https://github.com/4JX/L5P-Keyboard-RGB

Please integrate RGB control like this. Specifically smoother, customisable waves.

**Comment by 4JX** [comment]
I believe you might be sending this to the wrong repo

**Reply by xavier-phillips** [reply]
Yeah I meant to send it to this one https://github.com/BartoszCichecki/LenovoLegionToolkit

Is there any chance you could collaborate with the person behind this? It would be nice to have one integrated software for all legion things, and I really love your RGB software effects.
